src/common/btree.c

```diff
@@ -1194,7 +1197,9 @@ btr_node_split_and_insert(struct btr_context *tcx, struct btr_trace *trace,
 		D_DEBUG(DB_TRACE, "Splitting leaf node\n");
 
 		btr_rec_copy(tcx, rec_dst, rec_src, nd_right->tn_keyn);
-		btr_node_insert_rec_only(tcx, trace, rec);
+		rc = btr_node_insert_rec_only(tcx, trace, rec);
+		if (rc)
+			return rc;
```

Comment:
[Question] Should we release `off_right` somehow? Or is this intentionally ignored in this patch (as noted in the commit messsge)?

Reply:
Right, I didn't intend to fix the memory leak for VMEM tree in this PR.

Reply:
OK.